### ISSUE TITLE
'module' object has no attribute 'decompress' Recovering

### ISSUE THREAD

**Issue by lfmci** [eclipse-sumo/sumo#14950]
When generating a scenario with OSM Web Wizard it fails with this error ''module' object has no attribute 'decompress' Recovering'

**Comment by m-kro** [comment]
Which python version do you use? Is it still Python 2.x?

**Comment by lfmci** [comment]
2.7 .. Should it be 3.x

**Comment by m-kro** [comment]
Yes, the compress method was added to the python gzip module in Python 3.2
